src/firebase/dynamicLinks.ts

```diff
@@ -19,21 +21,26 @@ const commonDynamicLinkParams: Omit<FirebaseDynamicLinksTypes.DynamicLinkParamet
 }
 
 export async function createInviteLink(address: string) {
+  const { externalLinks } = getDynamicConfigParams(DynamicConfigs[StatsigDynamicConfigs.APP_CONFIG])
   return dynamicLinks().buildShortLink({
     ...commonDynamicLinkParams,
-    link: `${WEB_LINK}share/${address}`,
+    link: `${externalLinks.baseUrl}share/${address}`,
```

Comment:
In case we omit the final slash
```suggestion
    link: appendPath(externalLinks.baseUrl, `share/${address}`),
```